mysql-test/suite/rpl/t/rpl_parallel_sbm.test

```diff
@@ -65,8 +61,29 @@ if (`SELECT $sbm_trx1_arrive > ($seconds_since_idling + 1)`)
 }
 --echo # ..done
 
+--echo # MDEV-32265. If a replica is actively delaying a transaction when
+--echo # restarted, when the sql thread is back up, Seconds_Behind_Master
+--echo # should pick back up where it it left-off.
+--source include/stop_slave.inc
+--source include/start_slave.inc
```

Comment:
I'd love to see
``` Seconds_Behind_Master = SQL_Delay  - SQL_Remaining_Delay ```
that holds at this point, but I don't insist esp not being sure if any round-up can come to play.
Yet you can give it some though.

Reply:
I think this would result in an unstable test, as `query_get_value()` would run separate queries to get each field. Perhaps we can implement `query_get_values()` and this would be our first refactor with it :)